spec/services/record_declarations/retained/early_career_teacher_spec.rb

```diff
@@ -71,4 +71,13 @@ def generate_raw_event(params)
       expect { described_class.call(params.merge(declaration_type: "invalid")) }.to raise_error(ActionController::ParameterMissing)
     end
   end
+
+  context "when declaration date is invalid" do
+    it "raises a ParameterMissing error" do
+      params = ect_params.merge({ declaration_date: "2021-06-21 08:46:29" })
+      params[:raw_event] = generate_raw_event(params)
+      expected_msg = "param is missing or the value is empty: [\"The property '#/declaration_date' must be a valid RCF3339 date\"]"
```

Comment:
I think `ApiController` is changing the actual response to be more like 
```
"{\"bad_or_missing_parameters\":\"The property '#/declaration_date' must be a valid RCF3339 date\"}"
```

Writing down the message might confuse someone into thinking that this is what our api outputs look like.

Reply:
good point, we can change to match a substring

Reply:
Like [here](https://github.com/DFE-Digital/early-careers-framework/blob/2caff755dec2ae3b8ca851e063373659e53d9f88/spec/services/record_declarations/started/early_career_teacher_spec.rb#L84)